package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-native-immutable-list-view",
   "version": "0.4.5",
   "description": "A drop-in replacement for React Native's ListView, FlatList, and VirtualizedList.",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
`lib` is actually created during compilation, see [package.json scripts](https://github.com/cooperka/react-native-immutable-list-view/blob/master/package.json#L10-L17) for details. Please revert.